Wikipedia/Code/YearInReviewCoordinator.swift

```diff
@@ -928,7 +880,7 @@ final class YearInReviewCoordinator: NSObject, Coordinator {
     }
 
     private func needsPostSurveyLoginPrompt() -> Bool {
-        return !dataStore.authenticationManager.authStateIsPermanent && WMFDeveloperSettingsDataController.shared.showYiRV2
+        return !dataStore.authenticationManager.authStateIsPermanent && WMFDeveloperSettingsDataController.shared.showYiRV3
```

Comment:
Actually I think this post-survey login prompt will go away entirely - we're front-loading the login prompt for V3. I'll add a commit removing it.